submissions/Olha3001/Poppy/index.html

```diff
@@ -0,0 +1,102 @@
+<!DOCTYPE html>
```

Comment:
By default your popup is open, it must be closed, also, fix the order of interactive elements.